src/components/Img/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, BoxProps } from '@mui/material';
+import { useState } from 'react';
+
+export interface ImgProps extends BoxProps<'img'> {
+	fallback?: BoxProps<'img'>['src'];
+}
+
+/**
+ * This component will display an image given an src.
+ * The `fallback` prop has been added to make it easier to update the image in case the primary src fails.
+ */
+export const Img = ({ src: srcProp, fallback, ...props }: ImgProps) => {
```

Comment:
```suggestion
	src: srcProp,
	fallback,
	onerror,
	...props
```

```
onerror={() => {
				if (fallback != null) {
					setSrc(fallback);
				}
				onerror?.()
			}}
```

Reply:
sorry the diff is not right, in any case, what about desctructuring onerror and passing it to the onerror function ? just in case we need to do something else.